authenticator.js

```diff
@@ -81,6 +82,13 @@ AuthenticateGithub.prototype.getAuthorizationToken = function(username, password
       if (err) reject(err);
       else resolve(res.token);
     });
+  }).then(this.githubOrg && function(token) {
+    return new Promise(function(resolve, reject) {
+      github.orgs.getMember({ user: username, org: _this.githubOrg }, function(err, res) {
```

Comment:
based on this logic, I gather an error is returned if a user is not a member of the organization? You've confirmed this behavior hitting the live API?